content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Octopus Deploy
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
+---
+
+[Octopus Deploy](https://octopus.com) is a deployment automation server, designed to make it easy to orchestrate releases and deploy applications, whether on-premises or in the cloud. Octopus is unlike a traditional CI system. In fact, Octopus is not a CI system. However, it can integrate with your existing build pipeline and integrate with several of your existing CI systems, like Jenkins, TeamCity, Azure DevOps etc.
+
+## Prerequisites
+
+- A working installation of Octopus or a hosted instance from https://octopus.com.
+- An account on the [Pulumi Console](https://app.pulumi.com).
+- The latest version of Pulumi. Installation instructions are [here]({{< relref "/docs/get-started/install" >}}).
+- Setup a new project and [stack]({{< relref "/docs/intro/concepts/stack" >}}) using one of our [Get Started]({{< relref "/docs/get-started" >}}) guides or simply by running [`pulumi new`]({{< relref "/docs/reference/cli/pulumi_new.md" >}})
+and choosing one of the many templates that are available.
+- A bare repo and set the remote URL to be your GitHub project.
+
+## Sample Project
+
+For the sake of this walkthrough, we will try to deploy the simple AWS example located [here](https://github.com/pulumi/examples/tree/master/aws-ts-hello-fargate). The example deploys a containerized Python Flask app on [AWS Fargate](https://aws.amazon.com/fargate/). This example also shows a pattern that is common with Pulumi -- keeping your infrastructure app (Pulumi) with your actual application code, but certainly not necessary. Regardless of how you decide to structure your project, you will need to provide the package to Octopus somehow.
+
+Alternatively, you can also run `pulumi new [template]` to create a template project.
```

Comment:
You also mention the ability to run `pulumi new` earlier, so I'd actually just cut this paragraph for clarity. And just have the sample project you link too.

Are there some commands to run to set up this project?

e.g.

```bash
git clone ...
cd aws-ts-hello-fargate

# Create your Pulumi stack.
pulumi stack init ...
```

Reply:
Yep. Will rephrase this.